pipeline/tasks/deploy-code.yml

```diff
@@ -0,0 +1,63 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  AUTH_KEY: ((gcp.service-account-key))
+  PROJECT: ((gcp.project-name))
+  GCP_SECRETS_VERSION: ((gcp.secrets-version))
+  NETLIFY_BUCKET_NAME: ((gcp.netlify-bucket-name))
+
+inputs:
+- name: census-worth-self-help
+- name: content-repo
+
+caches:
+- path: census-worth-self-help/site/node_modules
+- path: .npm
+- path: .cypress-cache
+
+run:
+  path: sh
+  args:
+  - -exec
+  - |
+    set +x
+    echo $BACKEND_CONTENT_REPO
+    echo $AUTH_KEY > encrypted_key.txt
+    set -x
+    base64 -d encrypted_key.txt > service-account.json
+    gcloud auth activate-service-account --key-file service-account.json
+    gcloud config set project $PROJECT
+
+    mkdir netlify-cms-app
+    gsutil cp gs://$NETLIFY_BUCKET_NAME/worth-netlify-cms-app-2-9-1-x.tar.gz .
+    tar -xvzf worth-netlify-cms-app-2-9-1-x.tar.gz -C netlify-cms-app
+
+    apk add --no-progress --update nodejs nodejs-npm
+    echo "npm version"
+    npm -v
+
+    export ROOT_FOLDER=$(pwd)
+    export CYPRESS_CACHE_FOLDER=${ROOT_FOLDER}/.cypress-cache
+    cd census-worth-self-help/site
+    npm install --only=prod
+    npm install ../../netlify-cms-app --only=prod
```

Comment:
If we were to reuse this task for dev build we could this netlify CMS patch as an option (since for the dev build not much value in patched CMS) ... and we'd need to parametrise where we are getting the content repository from (since it's at ../content/ not content-repository), perhaps use an input parameter that specifies the location relative to ROOT_FOLDER where the content is.